charts/datadog/templates/NOTES.txt

```diff
@@ -255,3 +255,27 @@ helm install ksm https://charts.helm.sh/stable/packages/kube-state-metrics-2.9.1
 You are using datadog.kubeStateMetricsCore.enabled but you disabled the cluster agent. This configuration is unsupported and the kube-state-metrics core check can't be configured.
 To enable it please set clusterAgent.enabled to 'true'.
 {{- end }}
+
+{{- $hasContainerInclude := false }}
+{{- range $key := .Values.datadog.env }}
```

Comment:
maybe we should also check for the presence of `.Values.datadog.containerInclude` as well:
https://github.com/DataDog/helm-charts/blob/5056593ac4f07a463f48378f6d90a703a0279646/charts/datadog/templates/_containers-common-env.yaml#L67